modules/storage-create-storage-class.adoc

```diff
@@ -31,6 +31,13 @@ persistent volumes.
 .. Select the reclaim policy.
 
 .. Select `{Provisioner}` from the drop down list.
++
+ifeval::["{Provisioner}" == "kubernetes.io/aws-ebs"]
+[NOTE]
+====
+To create the storage class with the equivalent CSI driver, select `{CsiDriver}` from the drop down list. For more details, check xref:../persistent_storage/persistent-storage-aws.adoc#additional-resources[Additional Resources].
```

Comment:
- s/drop down list/drop-down list/ (for clarity, because it is a compound adjective)
- Would you mind also making this update (drop-down) in line 33?
- s/check/see/ (I think this is more often how we phrase xrefs)

Reply:
done